docs/internals/security.rst

```diff
@@ -200,10 +201,11 @@ between clients, which is handled through the repository. Therefore in
 a multiple-client scenario a repository can trick a client into
 reusing counter values by ignoring counter reservations and replaying
 the manifest (which will fail if the client has seen a more recent
-manifest or has a more recent nonce reservation). If the repository is
-untrusted, but a trusted synchronization channel exists between
-clients, the security database could be synchronized between them over
-said trusted channel. This is not part of Borg's functionality.
+manifest or has a more recent nonce reservation).
+
+If the repository is untrusted, there *must* exists trusted channel
+between clients and security databases *must* be synchronized with the
+outlined protocol. This is not part of Borg's functionality.
```

Comment:
Well, if we don't have it, it doesn't help to talk about that it must exist.

Reply:
I just took the comments in the issue and placed them here. If you think we should not change this phrase, I'll revert it.

Reply:
Yes, please revert.